packages/react-components/src/components/abstract-output-component.tsx

```diff
@@ -50,6 +47,8 @@ export interface AbstractOutputState {
 
 export abstract class AbstractOutputComponent<P extends AbstractOutputProps, S extends AbstractOutputState> extends React.Component<P, S> {
 
+    static HANDLE_WIDTH = 30;
```

Comment:
My reasoning was that the creator of the outputs should only be concerned with each output's total bounds, but how the output splits its own area would be internal to the output itself. It was still hardcoded but the constant was moved from the trace context to the component so that it didn't need to be passed around.

The handle width constant was public because it is used by the trace context to calculate the margin for the axis and navigator components. But I think this could be avoided if the sash offset is made to be relative to the output's left bound (including the handle).

Reply:
The handle width, Y-axis width and sash width are now optionally configurable through the props. Their default width values are internal to the output component.